components/TaskCard/TaskCard.js

```diff
@@ -1,26 +1,151 @@
+            name="title"
+            minlength="3"
+            maxlength="30"
+            placeholder="Enter a title"
+            required
+            value={taskout.title}
+            onChange={handleChange}
+          />
+
+          <StyledLabel htmlFor="description">Description:</StyledLabel>
+          <TextareaForDescription
+            id="description"
+            type="text"
+            name="description"
+            minlength="3"
+            maxlength="100"
+            placeholder="Enter a description"
+            required
+            value={taskout.description}
+            onChange={handleChange}
+          ></TextareaForDescription>
+
+          <StyledLabel htmlFor="prio">Priority:</StyledLabel>
+          <StyledSelect id="prio" name="prio" required onChange={handleChange}>
+            <option selected={taskout.prio === "Urgent"} value="Urgent">
+              Urgent
+            </option>
+            <option selected={taskout.prio === "Medium"} value="Medium">
+              Medium
+            </option>
+            <option selected={taskout.prio === "Low"} value="Low">
+              Low
+            </option>
+          </StyledSelect>
+
+          <StyledLabel htmlFor="dueDate">Due Date:</StyledLabel>
+          <StyledInput
+            type="date"
+            id="dueDate"
+            name="dueDate"
+            required
+            value={taskout.dueDate}
+            onChange={handleChange}
+          />
+
+          <StyledLabel htmlFor="category">Category:</StyledLabel>
+          <StyledSelect
+            id="category"
+            name="category"
+            required
+            onChange={handleChange}
+          >
+            <option
+              selected={taskout.category === "Tasks in Backlog"}
+              value="Tasks in Backlog"
+            >
+              Backlog
+            </option>
+            <option
+              selected={taskout.category === "Tasks in Progress"}
+              value="Tasks in Progress"
+            >
+              In Progress
+            </option>
+            <option
+              selected={taskout.category === "Awaiting Feedback"}
+              value="Awaiting Feedback"
+            >
+              Awaiting Feedback
+            </option>
+            <option selected={taskout.category === "Done"} value="Done">
+              Done
+            </option>
+          </StyledSelect>
+          <SaveTaskButton type="submit" onClick={handleSubmit}>
+            Save
+          </SaveTaskButton>
+        </Card>
+      ) : (
+        <Card>
+          <Title>{taskout.title}</Title>
+          <Description>{taskout.description}</Description>
+          <Priority>
+            <strong>Priority:</strong> {taskout.prio}
+          </Priority>
+          <DueDate>
+            <strong>Due Date:</strong> {taskout.dueDate}
+          </DueDate>
+          <Category>
+            <strong>Category:</strong> {taskout.category}
+          </Category>
```

Comment:
I think a way for better semantic here is a [description list element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl).